src/picongpu/include/particles/ionization/None/None.def

```diff
@@ -27,9 +29,17 @@ namespace particles
 namespace ionization
 {
 
+    template<typename T_DestSpecies, typename T_SrcSpecies = bmpl::_1>
```

Comment:
@n01r Could you write a comment on why you are using `T_SrcSpecies = bmpl::_1` here? (*chicken-and-egg problem*)